### ISSUE TITLE
Missing Logs Sysmon

### ISSUE THREAD

**Issue by Hestat** [wazuh/wazuh-ruleset#520]
I have been working to update a [ruleset](https://github.com/Hestat/ossec-sysmon) I manage to the latest wazuh version and while I've been able to get most of it figured out I was trying to update my parent -> child alerts for Word spawning powershell or command line and I am finding that the sysmon events are not making it to wazuh for processing while subsequent data is (like the powershell network connections). 

I have turned on logall, and edited rules to report all sysmon_event1 events but nothing is being pulled for the attached log. Using Wazuh v 3.10.2 both for server and client.
![missing-log](https://user-images.githubusercontent.com/22892300/68603304-08e37d80-0476-11ea-9701-8a91e93f2379.png)

Any thoughts or direction on why these events would not be reported?

**Comment by cristgl** [comment]
Hello @Hestat,

Let's try to find out the root cause of this issue.

1. Have you added a similar or the following block to your configuration?

```
<localfile>
  <location>Microsoft-Windows-Sysmon/Operational</location>
  <log_format>eventchannel</log_format>
</localfile>
```

2. After enabling the `logall` option, have you seen any Sysmon events at your _archives.log_ file?
3. Which Windows version are you using?
4. Do you have any warnings or errors at your _ossec.log_ file (in both manager and agent)?

Kind regards,
Cristina

**Comment by Hestat** [comment]
Hello thanks for responding,
Yes Symon log is in agent.conf
OS Data:Microsoft Windows 10 Pro 10.0.17763

I started to write up all the info, but then found an error in the ossec.log on the server. It looks like the obfuscation used in the malicious powershell broke the XML parsing. Not quite sure what can be done about that. Let me know what you think
![wazuh-trouble3](https://user-images.githubusercontent.com/22892300/68631434-e3c42e80-04b8-11ea-90bf-224abc1db1e4.png)

**Comment by cristgl** [comment]
Hi again, @Hestat,

This issue was reported in https://github.com/wazuh/wazuh/issues/3343 and fixed here https://github.com/wazuh/wazuh/pull/3320.
The problem is that the character " followed by , were being taken into account as the end of the event part of the log, this results in an invalid XML log which cannot be processed nor passed to JSON.
The fix will be ready for Wazuh version 3.11, I am so sorry for the inconveniences.

Kind regards,
Cristina

**Comment by cristgl** [comment]
Hi @Hestat,

I am closing this issue as the fix is already merged into branch 3.11, if you need further help, do not hesitate to ask again.

Kind regards,
Cristina
